### DISCUSSION TITLE
setCursor and printf to stay in column

### DISCUSSION THREAD

**Discussion by gert4git** [Bodmer/TFT_eSPI#2039] [Q&A - General]
Hello All,

I have this code that sets the cursor and then prints some stuff.

```
#include <TFT_eSPI.h> 
...
tft.setCursor(2*CX+20, 0, 4);
tft.printf("%02d:%02d:%02d\n", hh, mm, ss);
tft.printf("%04d-%02d-%02d", yr, mo, dy);
```
I expected the second line to follow the column position of the first. I.e. all at the same indentation or x-offset. But it does not. The second line jumps back to column 0.

Is there code to print multiple lines at the same X indentation? 

Cheers,
Gert

**Comment by Bodmer** [comment]
```
tft.setCursor(2*CX+20, 0, 4);
tft.printf("%02d:%02d:%02d", hh, mm, ss);
tft.setCursor(2*CX+20, 0 + tft.fontHeight(4), 4);
tft.printf("%04d-%02d-%02d", yr, mo, dy);
```

**Reply by gert4git** [reply]
@Bodmer,

Thanks. That is a doable approach.

Cheers,
Gert
